ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -62,6 +62,9 @@
         </S.Generic>
         {{! TODO: Target type filter}}
       </Hds::SegmentedGroup>
+      {{#if this.hasFilters}}
```

Comment:
This could actually be moved to inside the component I believe. And we could just check `{{#if this.filters}}` for the component

Reply:
Right. Good call.